[2022-08-19 21:20:16,389] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_aguardando scheduled__2022-08-19T21:15:00+00:00 [queued]>
[2022-08-19 21:20:16,580] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_aguardando scheduled__2022-08-19T21:15:00+00:00 [queued]>
[2022-08-19 21:20:16,581] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 21:20:16,581] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-19 21:20:16,581] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 21:20:16,688] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): popula_elastic_aguardando> on 2022-08-19 21:15:00+00:00
[2022-08-19 21:20:16,779] {standard_task_runner.py:52} INFO - Started process 93605 to run task
[2022-08-19 21:20:16,789] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'elastic_desk', 'popula_elastic_aguardando', 'scheduled__2022-08-19T21:15:00+00:00', '--job-id', '721', '--raw', '--subdir', 'DAGS_FOLDER/etl_elastic_desk.py', '--cfg-path', '/tmp/tmptox3ua_z', '--error-file', '/tmp/tmpzpiccy32']
[2022-08-19 21:20:16,791] {standard_task_runner.py:80} INFO - Job 721: Subtask popula_elastic_aguardando
[2022-08-19 21:20:17,634] {task_command.py:371} INFO - Running <TaskInstance: elastic_desk.popula_elastic_aguardando scheduled__2022-08-19T21:15:00+00:00 [running]> on host 1e1070d9e5ae
[2022-08-19 21:20:18,442] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clecio.antao@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=elastic_desk
AIRFLOW_CTX_TASK_ID=popula_elastic_aguardando
AIRFLOW_CTX_EXECUTION_DATE=2022-08-19T21:15:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-19T21:15:00+00:00
[2022-08-19 21:20:18,482] {base.py:271} INFO - GET http://192.168.2.100:9200/ [status:200 request:0.037s]
[2022-08-19 21:20:21,076] {base.py:271} INFO - HEAD http://192.168.2.100:9200/_template/desk_aguardando [status:200 request:0.019s]
[2022-08-19 21:20:21,366] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/es_pandas/es_pandas.py:58 RuntimeWarning: invalid value encountered in double_scalars
[2022-08-19 21:20:21,815] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.113s]
[2022-08-19 21:20:21,879] {python.py:173} INFO - Done. Returned value was: None
[2022-08-19 21:20:22,031] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=elastic_desk, task_id=popula_elastic_aguardando, execution_date=20220819T211500, start_date=20220819T212016, end_date=20220819T212022
[2022-08-19 21:20:22,215] {local_task_job.py:221} WARNING - State of this instance has been externally set to success. Terminating instance.
[2022-08-19 21:20:22,218] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 93605. PIDs of all processes in the group: [93605]
[2022-08-19 21:20:22,218] {process_utils.py:80} INFO - Sending the signal Signals.SIGTERM to group 93605
[2022-08-19 21:20:22,268] {process_utils.py:75} INFO - Process psutil.Process(pid=93605, status='terminated', exitcode=0, started='21:20:15') (93605) terminated with exit code 0
